### ISSUE TITLE
Device support for CYD-ESP32-2432S032

### ISSUE THREAD

**Issue by ElPolloDiabloGDC** [BruceDevices/firmware#593]
Hy,

is it possible to run bruce on this device as well? 
Is it enough to just change /ports/_New-Device-Model/platformio.ini and what exactly?

https://github.com/kpeeem/3.2inch_ESP32-2432S032

best regards

**Comment by bmorcelli** [comment]
This device uses GT911 touch driver... this driver isn´t supported yet..

**Comment by wertsaco** [comment]
> Hy,
> 
> is it possible to run bruce on this device as well? Is it enough to just change /ports/_New-Device-Model/platformio.ini and what exactly?
> 
> https://github.com/kpeeem/3.2inch_ESP32-2432S032
> 
> best regards

Hi, did you manage to make it run in ESP32-2432S032C???

**Comment by michaelolesen77-eng** [comment]
yes any news?  i have this model and the 4" both not supported yet :(
